wasm/doom/README.md

```diff
@@ -0,0 +1,25 @@
+# WASM Port of Doom game in browser
+
+This is a port of the [Doom game](https://en.wikipedia.org/wiki/Doom_(1993_video_game)) to the web using [WebAssembly](https://webassembly.org/).
+
+## Podman Compose
+
+To run the game, you need to have [Podman](https://podman.io/) installed, and the [`podman-compose`](https://github.com/containers/podman-compose) script available (included in Podman Desktop).
```

Comment:
should not link to `podman-compose` as we are prefering `podman compose` now using the compose binary